plugins/inputs/exec/exec.go

```diff
@@ -149,6 +155,9 @@ func (e *Exec) SetParser(parser telegraf.Parser) {
 
 func (e *Exec) Gather(acc telegraf.Accumulator) error {
 	var wg sync.WaitGroup
+
+	e.runner.SetDebug(e.Debug)
```

Comment:
Please do this in `Init()` **once** instead of setting the same value in each gather cycle.

Reply:
Done!